classes/class-wc-connect-shipping-label.php

```diff
@@ -209,9 +175,17 @@ public function get_selected_packages( WC_Order $order ) {
 							$formatted = WC_Connect_Compatibility::instance()->get_formatted_variation( $product, true );
 							$product_data['attributes'] = $formatted;
 						}
+						$customs_info = get_post_meta( $product_data['product_id'], 'wc_connect_customs_info', true );
```

Comment:
I ran into an issue with this: When I add a tariff number for a product, I see it saved in the db in post_meta, but this information is not loaded on subsequent label purchases with that product. This happens on both Calypso and wp-admin.

Reply:
I can't reproduce that bug. Are you sure the product was the same? The tariff number is per-variation, not per-product (because people use variations for all sorts of things, it's conviebable that different variations of the same product have a different code).

Reply:
I think I see why - the customs info only gets added if there are packages returned from `get_packaging_metadata()` in https://github.com/Automattic/woocommerce-services/blob/ec8db8844c05f33d44d66b9de2077bc48727c839/classes/class-wc-connect-shipping-label.php#L142

If there are none, then we send the default box, where the customs info isn't added:

https://github.com/Automattic/woocommerce-services/blob/ec8db8844c05f33d44d66b9de2077bc48727c839/classes/class-wc-connect-shipping-label.php#L147

So, we should add the customs info in there too.

Reply:
Fixed in 806e582, thanks for hunting this bug :)

Reply:
The fix works 👍